python/pylibwholegraph/CMakeLists.txt

```diff
@@ -17,23 +17,17 @@
 cmake_minimum_required(VERSION 3.26.4 FATAL_ERROR)
 
 include(../../rapids_config.cmake)
-set(WHOLEGRAPH_VERSION "${RAPIDS_VERSION_MAJOR_MINOR}.00")
-
-include(FetchContent)
-
-FetchContent_Declare(
-        rapids-cmake
-        GIT_REPOSITORY https://github.com/rapidsai/rapids-cmake.git
-        GIT_TAG        origin/branch-${RAPIDS_VERSION_MAJOR_MINOR}
-)
-FetchContent_MakeAvailable(rapids-cmake)
 include(rapids-cmake)
 include(rapids-cuda)
 include(rapids-cpm)
 
 rapids_cuda_init_architectures(PYLIBWHOLEGRAPH)
 
-project(PYLIBWHOLEGRAPH VERSION ${WHOLEGRAPH_VERSION} LANGUAGES CXX CUDA)
+project(
+  PYLIBWHOLEGRAPH
```

Comment:
We're a little inconsistent about this across repos but it might be nice to use lowercase `pylibwholegraph` here. However, it's fine if that is out-of-scope for this PR (and/or not important enough to change at all).

e.g. cuDF has `project(CUDF...)` but also `project(pylibcudf...)`, so our casing is not consistent even within a repo.

Reply:
Thanks, yeah I'd noticed that. Let's treat it as out-of-scope for this PR. I'd like to merge this to simplify things ahead of setting up `libwholegraph` wheels.

The project name flows into a bunch of other variables that CMake generates (see https://cmake.org/cmake/help/latest/command/project.html#command:project) and I'm not sure if those are case-sensitive, so changing the case could require other changes to CMake code here (and maybe in other places that build wholegraph from source, like RAPIDS devcontainers). I'd rather not work through that right now.

Reply:
Yes, all those downstream uses are case sensitive. At some point we may want to strive for consistency across RAPIDS but it’s low priority - lots of work, risk of breakage, and not a lot of value.